core/src/scss/utilities/variables/core/_core-color-maps.scss

```diff
@@ -7,40 +7,40 @@
 // Color maps for individual components are in their own respective directories within utilities/variables/components
 //
 // Markup: $core-colors: (
-//    brand:                        $color-cardinal-red,
-//    brand--bright:                $color-bright-red,
-//    brand--dark:                  $color-dark-red,
-//    bg:                           $color-white,
-//    bg--reverse:                  $color-black,
-//    text:                         $color-black,
+//    brand:                        $su-color-cardinal-red,
+//    brand--bright:                $su-color-bright-red,
+//    brand--dark:                  $su-color-dark-red,
+//    bg:                           $su-color-white,
+//    bg--reverse:                  $su-color-black,
+//    text:                         $su-color-black,
 //    text--high-contrast:          #000,
-//    text--reverse:                $color-white,
-//    link:                         $color-bright-blue,
-//    link--hover:                  $color-black,
-//    link--alt:                    $color-bright-red,
-//    link--alt-hover:              $color-black,
-//    link--reverse:                $color-white,
-//    link--reverse-hover:          $color-vivid-red,
+//    text--reverse:                $su-color-white,
+//    link:                         $su-color-bright-blue,
+//    link--hover:                  $su-color-black,
+//    link--alt:                    $su-color-bright-red,
+//    link--alt-hover:              $su-color-black,
+//    link--reverse:                $su-color-white,
+//    link--reverse-hover:          $su-color-vivid-red,
 //) !default;
 //
 // Style guide: Variables.Core.colors-map
 //
 $core-colors: (
```

Comment:
I believe we need to change this to $su-core-colors and add the deprecated message for it also.